src/components/Quiz/Quiz.tsx

```
@@ -0,0 +1,511 @@
+});
+
+const QuizAlternative = () => {
+  const {
+    getCurrentQuestion,
+    selectAnswer,
+    getCurrentAnswer,
+    isQuestionSkipped,
+  } = useQuizStore();
+  const currentQuestion = getCurrentQuestion();
+  const currentAnswer = getCurrentAnswer();
+  const isCurrentQuestionSkipped = currentQuestion
```

ESLint

[error] 'isCurrentQuestionSkipped' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)